app/src/main/java/org/p2p/wallet/swap/jupiter/statemanager/SwapStateManager.kt

```diff
@@ -21,6 +23,10 @@ class SwapStateManager(
 
     fun observe(): StateFlow<SwapState> = state
 
+    suspend fun <T> getStateValue(getter: (state: SwapState) -> T): T {
```

Comment:
is it okay to get token a and b from state? 
@NikolaiEmelianov

Reply:
better to listen the Flow. 
If tokenA change you reinstall screen state
If catch some wrong state then you close screen or something
But in this local feature case Ok. I will disable click for wrong state

Reply:
click that open this screen